src/compiler.jl

```diff
@@ -6225,7 +6243,9 @@ struct EnzymeCompilerParams <: AbstractEnzymeCompilerParams
     # Whether to (in aug fwd) += by one
     shadowInit::Bool
     expectedTapeType::Type
+    method_table
```

Comment:
Do we need it here? Can we not just get it from the EnzymeTarget?

Reply:
per discussion, parent_job sadly